services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/secondary_feedback/prompt_builder.rb

```diff
@@ -20,14 +20,23 @@ class PromptBuilder < Evidence::GenAi::PromptBuilder
         private def default_template = DEFAULT_TEMPLATE
         private def template_folder = TEMPLATE_FOLDER
 
-        private def primary_secondary_examples = markdown_table_rows(feedback_data_tuples)
-        private def feedback_data_tuples = feedback_data.map { |f| [f.primary, f.secondary] }
+        private def primary_secondary_examples = markdown_table_rows(feedback_example_tuples)
+        private def feedback_example_tuples = feedback_examples.map { |f| [f.primary, f.secondary] }
```

Comment:
I like `pluck` here since it gets rid of the block and temp variables:
`private def feedback_example_tuples = feedback_examples.pluck(:primary, :secondary)`